numba/core/pylowering.py

```diff
@@ -137,12 +136,12 @@ def lower_inst(self, inst):
 
         elif isinstance(inst, ir.Branch):
             cond = self.loadvar(inst.cond.name)
-            if cond.type == Type.int(1):
+            if cond.type == llvmlite.ir.IntType(1):
```

Comment:
Might be nice in a subsequent PR to have a `llvm_consts` module or similar such that this sort of thing can be written as
```python
if cond.type == llvm_consts.i1:
```
I'm fairly sure the code base as such definitions made in many places.